src/lib.rs

```diff
@@ -142,6 +143,7 @@
 //! [20]: https://www.unix.com/man-page/mojave/4/random/
 //! [21]: https://www.freebsd.org/cgi/man.cgi?query=getrandom&manpath=FreeBSD+12.0-stable
 //! [22]: https://leaf.dragonflybsd.org/cgi/web-man?command=getrandom
+//! [23]: https://cygwin.com/git/?p=newlib-cygwin.git;a=blob;f=winsup/cygwin/include/sys/random.h;hb=HEAD#l22
```

Comment:
If we are depending on the esp targets, we should change the docs (and doc link) to point to:

https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/random.html